packages/commonwealth/client/scripts/views/components/react_quill_editor/react_quill_editor.tsx

```diff
@@ -236,51 +242,67 @@ const ReactQuillEditor = ({
-                    ? markdownKeyboardShortcuts
-                    : undefined,
-                  imageUploader: {
-                    upload: handleImageUploader,
-                  },
-                }}
-              />
-            </div>
+            <DragDropContext onDragEnd={handleDragStop}>
+              <Droppable droppableId="quillEditor">
+                {(provided) => (
+                  <div
+                    className={`${isDraggingOver ? 'ondragover' : ''}`}
+                    {...provided.droppableProps}
+                    ref={provided.innerRef}
+                    onDragOver={handleDragStart}
+                    onDragLeave={handleDragStop}
+                    onDrop={handleDragStop}
+                  >
+                    <div data-text-editor="name">
+                      <ReactQuill
+                        ref={editorRef}
+                        className={clsx('QuillEditor', className, {
+                          markdownEnabled: isMarkdownEnabled,
+                        })}
+                        placeholder={placeholder}
+                        tabIndex={tabIndex}
+                        theme="snow"
+                        bounds={`[data-text-editor="name"]`}
+                        value={contentDelta}
+                        onFocus={() => setIsFocused(true)}
+                        onBlur={() => setIsFocused(false)}
+                        onChange={handleChange}
+                        onChangeSelection={(selection: RangeStatic) => {
+                          if (!selection) {
+                            return;
+                          }
+                          lastSelectionRef.current = selection;
+                        }}
+                        formats={isMarkdownEnabled ? [] : undefined}
+                        modules={{
+                          toolbar: {
+                            container: `#${toolbarId}`,
+                            handlers: isMarkdownEnabled
+                              ? markdownToolbarHandlers
+                              : undefined,
+                          },
+                          imageDropAndPaste: {
+                            handler: handleImageDropAndPaste,
+                          },
+                          clipboard: {
+                            matchers: clipboardMatchers,
+                          },
+                          mention,
+                          magicUrl: !isMarkdownEnabled,
+                          keyboard: isMarkdownEnabled
+                            ? markdownKeyboardShortcuts
+                            : undefined,
+                          imageUploader: {
+                            upload: handleImageUploader,
+                          },
+                        }}
+                      />
+                    </div>
+                    {provided.placeholder}
+                  </div>
+                )}
+              </Droppable>
+            </DragDropContext>
```

Comment:
these commits should not be part of this PR

Reply:
Removed.